elements.tex

```diff
@@ -0,0 +1,50 @@
+\section{Elements of the Early Science Program}
+
+The Early Science Program consists of the following elements:
+\begin{itemize}
+	\item A series of three \emph{Data Previews} based on simulated LSST-like data and reprocessed data taken during the Rubin Observatory commissioning period with the LSST Science Camera (LSSTCam).
+	\item A world-public stream of alerts from transient, variable, and moving sources that will be scaled up during commissioning and the first year of the survey.
+	\item Template generation, both prior to the start of regular survey operations based on data collected during the commissioning period with LSSTCam, and incrementally during the first year regular survey operations  to maximize the number of templates available for Alert Production IN YEAR 1.
+%		\item Incremental generation of templates during the first year of survey operations to steadily increase Alert Production and distribution to Community Brokers over the course of the first year of the survey.
+	\item LSST Data Release 1 (DR1), which will be based on the Data Release Processing (DRP) of the first six months of LSST data.
+\end{itemize}
+
+% Data Previews
+% This provides early integration tests of our system, prepare the team to be operations-ready, and prepare the community to use Rubin data products and services
+
+\subsection{Data Previews}
+
+The twin goals of the Data Previews are 1) to familiarize the community with LSST analysis in advance of the first survey data release, and b) to help drive development of Rubin operations capabilities prior to survey start.
+The images for both Data Preview 1 (DP1) and Data Preview (DP2) will be taken with LSSTCam, since (as of November 2022) the Commissioning Camera (ComCam) is no longer scheduled to have any on-sky observation time.
+
+Three pre-survey data previews are planned:
+\begin{itemize}
+\item  Data Preview 0 (DP0): Based on simulated LSST-like data
+\item  Data Preview 1: Based on a few nights of early science-grade commissioning data taken with LSSTCam
+\item Data Preview 2: Based on a full reprocessing of all science-grade LSSTCam data.
+
+\end{itemize}
+
+% Rubin pre-operations is working with the construction project to provide early access to static-sky data products via Data Previews.
+The first two phases of Data Preview 0 (DP0) were released to a group of early adopters from the community in the summers of 2021 and 2022, and are based on the DESC DC2 simulated dataset, \citep{2021ApJS..253...31L}. A third phase, focusing on simulated solar system science catalogs, is planned for release in summer 2023.
+
+{\bf Add in a review of DP0.1, DP0.2 and an overview of  DP0.3  here.}
```

Comment:
```suggestion
```

Reply:
I think the text now contains enough review of DP0.

Reply:
That was an old todo that I had not removed